frontend/src/meeting-detail/RegistrationForm.jsx

```diff
@@ -94,6 +97,7 @@ export default function RegistrationForm({ isDisabled = false }) {
               value={CUNYfirstID}
               onChange={(e) => setCUNYfirstID(e.target.value)}
             />
+            {isShowValidated && CUNYfirstID.length !== 8 && isIdInvalid ?<div className="invalid">CUNYfirst ID should be 8 digit numbers.</div>:''}
```

Comment:
Hopefully you can write a complete validator to validate that the user can only enter 8 digits.
For example, inputting "1234567" or "abcdefgh" cannot pass.

Reply:
I have use 'isIdInvalid' function to check the user cuny id input, I think it works now.